### PR TITLE
Disabled UIPicker test

### PR DESCRIPTION
**Pull request by DiedeGu**
I have implemented @khandpur's advice as far as I could. 

1) I have added a Disabled UIPicker in the Interface builder on top of the others. It is intergrated within the same functions and uses the same DataSource and Delegate as the custom Picker (File Owner). 
Only within the Interface builder I have disabled the UIPicker. I could also do this only in code or do both. Please let me know what is required. 

2) I have added a test, to see if the UIPicker throws an error if disabled. I do not know if I am correctly checking if the Error is populated correctly. If you have a example test I am very willing to integrate it. 

3) I have added a constrain to the UIPickerAction as requested. 

Feel free to give me any advice or critic, this is my first pull request and I would like to learn. 

note: I have 3 failing tests at the Functional test, the WebViewTests. I did not change them so that is strange. All UnitTests ran successfully. 

note: the GREYPinchAction.m line 163 and 164 gave a build error, so I had to cast them to CGFloat. This is not part of my fix.

### CODE REVIEW

**Comment by DiedeGu** [comment]
I signed it!

**Comment by DiedeGu** [comment]
I signed it!

**Comment by DiedeGu** [comment]
I signed it!

File: Tests/FunctionalTests/TestRig/Sources/FTRActionSheetViewController.xib [hunk 1]
```diff
@@ -36,8 +36,8 @@
                     <color key="textColor" red="0.0" green="0.0" blue="0.0" alpha="1" colorSpace="calibratedRGB"/>
                     <nil key="highlightedColor"/>
                 </label>
-                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="rbH-eF-pyb">
```
Comment (tirodkar) [inline Tests/FunctionalTests/TestRig/Sources/FTRActionSheetViewController.xib]: Is this change needed?

File: Tests/FunctionalTests/TestRig/Sources/FTRAccessibilityViewController.xib [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="6254" systemVersion="13F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="9531" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES">
```
Comment (tirodkar) [inline Tests/FunctionalTests/TestRig/Sources/FTRAccessibilityViewController.xib]: Same here, I thought only the picker VC's xib would need to be modified?

**Comment by tirodkar** [comment]
Delete the xcworkspacedata and gcda files.

**Comment by tirodkar** [comment]
Also, @DiedeGu, can you please sign the CLA?

**Comment by khandpur** [comment]
Is the CLA actually signed? it should have been updated automatically once signed.

**Comment by tirodkar** [comment]
@khandpur, I don't think it is. @DiedeGu, could you please confirm?

**Comment by DiedeGu** [comment]
Hi @tirodkar and @khandpur the CLA is signed, I had to do it twice because the first time the email address of this account and the sign in didn't match. The second time the bot would not run. Any solutions?

**Comment by khandpur** [comment]
I beleive you need to redo the commit with the proper account that signed the CLA. Unfortunately without the CLA set to YES, we don't be able to merge this code :(

**Comment by bootstraponline** [comment]
The email on the commit doesn't match the email on file with the CLA bot. Amend the commit then force push to update the pull request.

**Comment by tirodkar** [comment]
@DiedeGu, were you able to look into this?

**Comment by tirodkar** [comment]
PING.

**Comment by DiedeGu** [comment]
PONG! Sorry for letting you guys wait, I am currently also looking into the reorganisation of my company so I haven't had the time. I can make a new commit tomorrow :).

**Comment by DiedeGu** [comment]
@tirodkar Hi Aditya, the tests that are failing are not connected to files I edited. I think this fix is oke. (I have no clue why the others are failing, the WebView also crashed at local runs). 

Warm Regards, 

Diede
